raft/log.go

```diff
@@ -1780,6 +1829,8 @@ func (l *Log) ReadFrom(r io.ReadCloser) error {
 		return nil
 	}
 
+	l.Logger.Printf("reading from stream")
```

Comment:
nm, I see a bunch of them below, assuming this is needed for debugging in the wild.